CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 cmake_minimum_required(VERSION 3.23)
 project(3DMMForever LANGUAGES CXX C)
+
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 
-set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
+set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
```

Comment:
I'm against making this change because it means the runtime needs to be installed from whatever version of VS2022 was used to build. Using it the other way allows us to just dump it into a zip file or installer and not have to do extra runtime magic.